src/groundlight/binary_labels.py

```diff
@@ -1,51 +1,98 @@
-"""Defines the possible values for binary class labels like Yes/No or PASS/FAIL.
+"""Defines the possible values for binary class labels like YES/NO.
 Provides methods to convert between them.
 
 This part of the API is kinda ugly right now.  So we'll encapsulate the ugliness in one place.
 """
 import logging
-from typing import List, Union
+from enum import Enum
+from typing import Any, Union
 
-from model import Detector, ImageQuery
+from model import ClassificationResult, Detector, ImageQuery
 
-logger = logging.getLogger("groundlight.sdk")
+logger = logging.getLogger(__name__)
 
 
-def internal_labels_for_detector(
-    context: Union[ImageQuery, Detector, str],  # pylint: disable=unused-argument
-) -> List[str]:
-    """Returns an ordered list of class labels as strings.
-    These are the versions of labels that the API demands.
-    :param context: Can be an ImageQuery, a Detector, or a string-id for one of them.
-    """
-    # NOTE: At some point this will need to be an API call, because these will be defined per-detector
-    return ["PASS", "FAIL"]
+class Label(str, Enum):
+    YES = "YES"
+    NO = "NO"
+    UNSURE = "UNSURE"
+
+
+VALID_DISPLAY_LABELS = {Label.YES, Label.NO, Label.UNSURE}
+
+
+class DeprecatedLabel(str, Enum):
+    PASS = "PASS"
+    FAIL = "FAIL"
+    NEEDS_REVIEW = "NEEDS_REVIEW"
+
+
+DEPRECATED_LABEL_NAMES = {DeprecatedLabel.PASS, DeprecatedLabel.FAIL, DeprecatedLabel.NEEDS_REVIEW}
 
 
 def convert_internal_label_to_display(
     context: Union[ImageQuery, Detector, str],  # pylint: disable=unused-argument
     label: str,
 ) -> str:
+    """Convert a label that comes from our API into the label string that we show to the user.
```

Comment:
I'm tempted to return enums instead of strings because our current service behavior does not support arbitrary strings, and it would steer users in the right direction. But I also understand that some users might not like enums, and prefer to work with strings.